### PR TITLE
Update VMWare ISO

### PR DESCRIPTION
**Pull request by Jason-ZW**
Update VMWare ISO

### CODE REVIEW

**Comment by Jason-ZW** [comment]
#2283

File: scripts/package-installer [hunk 1]
```diff
@@ -32,7 +32,11 @@ mkdir -p ${DIST}/boot/isolinux/
 cat scripts/isolinux.cfg | envsubst >         ${DIST}/boot/isolinux/isolinux.cfg
 cat scripts/isolinux_label.cfg | LABEL=${VERSION} envsubst >   ${DIST}/boot/linux-current.cfg
 #cat scripts/isolinux_label.cfg | LABEL=debug APPEND="rancher.debug=true" envsubst >   ${DIST}/boot/linux-previous.cfg
-cat scripts/global.cfg | LABEL=${VERSION} envsubst >   ${DIST}/boot/global.cfg
+if [ -n "$VMWAREAUTOFORMAT" ];then
```
Comment (niusmallnan) [inline scripts/package-installer]:
```
if [ "$VMWAREAUTOFORMAT" = 1 ]; then
```

File: Dockerfile.dapper [hunk 2]
```diff
@@ -85,6 +85,7 @@ ARG OS_BASE_URL_arm64=https://github.com/rancher/os-base/releases/download/v2018
 ARG SYSTEM_DOCKER_VERSION=17.06-ros4
 ARG SYSTEM_DOCKER_URL_amd64=https://github.com/niusmallnan/os-system-docker/releases/download/${SYSTEM_DOCKER_VERSION}/docker-amd64-${SYSTEM_DOCKER_VERSION}.tgz
 ARG SYSTEM_DOCKER_URL_arm64=https://github.com/niusmallnan/os-system-docker/releases/download/${SYSTEM_DOCKER_VERSION}/docker-arm64-${SYSTEM_DOCKER_VERSION}.tgz
+ARG VMWAREAUTOFORMAT=""
```
Comment (niusmallnan) [inline Dockerfile.dapper]: VMWARE_AUTOFORMAT=1

File: Makefile [hunk 3]
```diff
@@ -64,7 +64,7 @@ rpi64:
 	cd scripts/images/raspberry-pi-hypriot64/ \
 		&& ../../../.dapper
 
-vmware:
+vmware: default
```
Comment (niusmallnan) [inline Makefile]: Get rid of `default`

**Comment by niusmallnan** [comment]
LGTM

**Pull request merged**